portfolio.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="CSS/styles.css">
+  </head>
+  <body>
+
+  <nav>
+    <a href="index.html">Home</a>
+  </nav>
+
+   <img class="portfolio" src="Images/painting.jpg">
```

Comment:
This is a good opportunity to use the class for styling as opposed to the `img` tag. 🔨